index.html

```diff
@@ -24,7 +24,20 @@ <h1>Welcome to the student showcase!</h1>
         <div>FOUR</div>
         <div>FIVE</div>
         <div>SIX</div>
-        <div>SEVEN</div>
+        <div>
+          <h1>Hello my name is Stefan West</h1>
+          <h2>Im learning to Code</h2>
+          <img src="https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/28056792_2080444551972457_8122567461366686076_n.jpg?oh=6c318ad002e39c57932408f6f60c92da&oe=5B0F43F2" alt="mypic">
+          <h3><strong>"Always Remain Teachable"</strong></h3>
```

Comment:
We don't use `strong` because we know how to use `font-weight` instead. (Also `strong` won't do what you think it should do, here).